### DISCUSSION TITLE
Allowing unselecting Tabs

### DISCUSSION THREAD

**Discussion by timvandam** [mantinedev/mantine#673] [Feedback]
I am using Tabs to switch between views, but also want the possibility to hide the views altogether. For this, I would like to be able to click the currently active tab to unselect it.

If this is deemed a valuable addition to Mantine I can try creating a PR that adds a prop that enables this behavior. What y'all think about this?

**Comment by rtivital** [comment]
Nope, that will be an accessibility violation, see specs – https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html

**Reply by timvandam** [reply]
Interesting, thanks for the resource
